resources/dicts/events/lifegen_events/murder.json

```diff
@@ -428,7 +438,9 @@
         "The Warrior Code commands that elders are to be respected. And yet here v_c is, rolling their eyes whenever you speak, laughing about you behind your back. You've even heard them muttering with their friends about how useless you are! You politely ask v_c on a walk. When you return to camp, the other cats assume the broad smile on your lips are from the nice weather, not from the feeling of your claws tearing through v_c's pelt."
     ],
     "murder elder medicinecat": [
-        "As you visit the medicine cat's den for a check-up, you swipe deathberries off of one of the many shelves and hide them under your paws. You tell v_c that you're feeling under the weather, and they give you some feverfew for it. You eat it before spitting it out- screeching and accusing the medicine cat of poisoning it. v_c sighs and insists it's fine, and you slip deathberries into the remaining bundle before they eat it to prove it wasn't poisoned. You watch as they realize what's happening, and leave the den feeling smug about what you'd done."
+        "As you visit the medicine cat's den for a check-up, you swipe deathberries off of one of the many shelves and hide them under your paws. You tell v_c that you're feeling under the weather, and they give you some feverfew for it. You eat it before spitting it out- screeching and accusing the medicine cat of poisoning it. v_c sighs and insists it's fine, and you slip deathberries into the remaining bundle before they eat it to prove it wasn't poisoned. You watch as they realize what's happening, and leave the den feeling smug about what you'd done.",
+        "v_c is always treating you poorly, frankly, you like their apprentice better than them. The younger medicine cat would be greater for the clan anyways, so, you decide to take care of that for yourself.",
```

Comment:
purrobably can be reworded

"v_c is always treating you poorly - and frankly, you like their apprentice a lot more. The younger cat would be better for the Clan anyways, so you decide to take care of the matter yourself."